docs/source/welcome/language-for-grants.md

```diff
@@ -0,0 +1,41 @@
+
+Researchers affiliated with Northeastern must request an access account on the SDE through the Office of Research. Once approved, researchers can access the SDE through a multi-factor authentication process. Data transfer of secure data is completed through Globus, a data management system that has a higher assurance level to meet secure data compliance requirements
+
+
+## NU SDE
+
+The Secure Data Enclave (SDE) is a subset of RC systems configured to comply with Northeastern University's CMMC Programme, creating a storage and computational environment that allows researchers to compete for government funding without having to support their own computational environments. The SDE is targeted at computational research. There is no physical lab, there is no physical CUI, but there is a cluster of hardware dedicated to storing and computing on CUI data.
+
+## Storage Services
+
+The focus of any CMMC-approved environment is storage — specifically security, privacy, and auditing. The SDE has two (2) Infinidat systems that are configured with FIPS-140-2 validated encryption — one (1) at the MGHPCC in Holyoke, MA and the other in Boston, MA at the Markley Datacenter.
+
+These systems both provide snapshot facilities, but the system at Markley is a dedicated off-site replication target for MGHPCC, the primary SDE datastore.
+
+Data is staged into and out of the cluster using Globus ([www.globus.org](www.globus.org)), which provides the encryption and auditing required of CMMC environments.
+
+## Compute Services
+
+The SDE also offers compute services via Open OnDemand ([www.openondemand.org](www.openondemand.org)). OOD is a web-ui to an HPC cluster that presents interfaces to traditional batch processing, a conventional "desktop-style" research, and client-server based programs. The HPC cluster is managed by SLURM ([www.schedmd.com](www.schedmd.com)), making the SDE a conventional research cluster with the enhanced security and auditing required for CMMC.
+
+## Data Management Plan (DMP)
+
+The Research Computing Team at Northeastern University ([www.rc.northeastern.edu](https://www.rc.northeastern.edu/)) provides research groups with up to 35 TB of high-performance data storage free of charge. This storage is owned and maintained by the University and is physically located in the Massachusetts Green High Performance Computing Center, a secure data center. All storage systems are on uninterruptible power supply (UPS)-backed power supplies, so that in the event of a power outage or interruption, there can be a graceful shutdown, minimizing the possibility of data loss. All storage file systems are backed up several times daily, and in certain cases, a second offsite copy is maintained. For secure, reliable data access, the Research Computing team maintains a Globus endpoint on Discovery that can be used by researchers to access and share their data.

```

Comment:
```suggestion
The Research Computing Team at Northeastern University ([www.rc.northeastern.edu](https://www.rc.northeastern.edu/)) provides research groups with up to 35 TB of high-performance data storage free of charge. This storage is owned and maintained by the university and is physically located in the Massachusetts Green High Performance Computing Center, a secure data center. All storage systems are on uninterruptible power supply (UPS)-backed power supplies, so that in the event of a power outage or interruption, there can be a graceful shutdown, minimizing the possibility of data loss. All storage file systems are backed up several times daily, and in certain cases, a second offsite copy is maintained. The Research Computing team maintains a Globus endpoint on Discovery for secure, reliable data access, enabling researchers to access and share their data.
```